openlibrary/catalog/get_ia.py

```diff
@@ -172,7 +177,7 @@ def read_marc_file(part, f, pos=0):
     :rtype: (int, str, str)
     :return: (Next position, Current source_record name, Current single MARC record)
     """
-    for data, int_length in fast_parse.read_file(f):
+    for data, int_length in fp_read_file(f):
```

Comment:
Perhaps `fast_read_file` would be better since the first thing I think of when I see `fp` in this context is "file pointer."